codesystems/CodeSystem-medicationrequest-course-of-therapy.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CodeSystem xmlns="http://hl7.org/fhir">
+    <id value="346d5570-a30d-4447-8aa1-ad5fad36b025"/>
+    <url value="https://fhir.nhs.uk/CodeSystem/medicationrequest-course-of-therapy"/>
```

Comment:
CodeSystem URL needs update to confirm with naming convention

Reply:
updated URL